server/controllers/class/getClassStudents.ts

```diff
@@ -0,0 +1,14 @@
+import { NextFunction, Request, Response } from 'express';
+import { getClassStudentsQuery } from '../../queries/class';
+
+const getClassStudents = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const { classId } = req.params;
+    const data = await getClassStudentsQuery(classId);
```

Comment:
Try to throw an error if something happens and the data returned null, it happened to me!